cmd/configure.go

```diff
@@ -130,10 +127,6 @@ func init() {
 
 	viper.SetDefault("year", time.Now().Local().Year())
 	viper.SetDefault("day", time.Now().Local().Day())
-	viper.SetDefault("session-token", "<enter aoc token here>")
-	viper.SetDefault("root-dir", "<enter root directory for problems>")
-	viper.SetDefault("python-exec", "<enter path for python executable>")
-	viper.SetDefault("leaderboard", "<enter private leaderboard id>")
```

Comment:
These guys were messing with the "required checks" so my stuff failed with 500 when fetching leaderboard as it used `<enter private leaderboard id>` as id 😢

Reply:
Ah snyggt! Jag glömde ta bort dem. De passade bättre innan bubble gui kom in